python/kfserving/kfserving/transformer.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import List, Dict
+from .server import Protocol
+from .server import KFModel
+from .server import KFSERVER_LOGLEVEL
+from .server import PREDICTOR_URL_FORMAT
+from kfserving.protocols.tensorflow_http import TensorflowRequestHandler
+logging.basicConfig(level=KFSERVER_LOGLEVEL)
+
+
+class Transformer(KFModel):
```

Comment:
seems confusing that the transformer is a KFModel and includes methods called explain and predict?

Reply:
Explain is not implemented in transformer but it is an interface in the base, I can remove it just gives warning. We still need predict method since in transformer it is a client call to the predictor.